staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -894,6 +919,10 @@ func (qs *queueSet) finishRequestLocked(r *request) {
 		qs.totSeatsInUse -= r.MaxSeats()
 		metrics.AddRequestConcurrencyInUse(qs.qCfg.Name, r.fsName, -r.MaxSeats())
 		qs.execSeatsGauge.Add(-float64(r.MaxSeats()))
+		if qs.seatDemandIntegrator == nil {
+			klog.ErrorS(nil, "nil setDemandIntegrator", "QueueSet", qs.qCfg.Name)
```

Comment:
The line below would panic anyway - so please remove.

Reply:
done